eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -159,3 +149,23 @@ jobs:
       # Shut down proxy to prevent file locks on the log file for auto-injected credscan
       # steps on windows
       - template: /eng/common/testproxy/test-proxy-tool-shutdown.yml
+
+      - pwsh: |
+          $files = Get-ChildItem -Path $(Build.SourcesDirectory) -Filter test-proxy.log
+          foreach($file in $files){
+              Write-Host "##[group]$file"
+              cat $file
+              Write-Host "##[endgroup]"
+          }
+        displayName: Dump Test-Proxy Logs
+        condition: succeededOrFailed()
+
+#      - pwsh: |
```

Comment:
Do you plan to delete this?

Reply:
I'm planning on enabling this once eng/common supports Test Proxy dumping error logs as well:

https://github.com/Azure/azure-sdk-tools/pull/9460